doc/options.rst

```diff
@@ -20,7 +20,7 @@ Optional:
 
 Scenario
 ~~~~~~~~
-The scenario-object is used to configure *SMAC* and can be constructed either by providing an actual
+The `scenario-object <apidoc/smac.scenario.scenario.html>`_ is used to configure *SMAC* and can be constructed either by providing an actual
```

Comment:
I think it would be better to reference the class if possible, something like:

    :class:`smac.scenario.scenario.Scenario`

as this is the entity present in SMAC (while the html is only rendered).

Reply:
I was not aware of that possibility. Very neat, thank you for pointing that out.